index.html

```diff
@@ -16,45 +16,23 @@
-          <img class="logo" src="./img/logo.png" alt="logo" />
+          <img class="logo" src="img/logo.png" alt="logo" />
         </a>
-        <nav>
-          <ul class="nav-bar-list flex">
-            <li>
-              <a href="#portfolio" class="nav-item">Portfolio</a>
-            </li>
-            <li>
-              <a href="#about-me" class="nav-item">About</a>
-            </li>
-            <li>
-              <a href="#contact-section" class="nav-item">Contact</a>
-            </li>
-            <li>
-              <a href="#" class="nav-item">
-                <svg
-                  width="24"
-                  height="24"
-                  viewBox="0 0 24 24"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path
-                    fill-rule="evenodd"
-                    clip-rule="evenodd"
-                    d="M7.4 5H16.6C17.7531 5 18.4977 5.00156 19.0643 5.04785C19.6077 5.09225 19.8091 5.1676 19.908 5.21799C20.2843 5.40973 20.5903 5.71569 20.782 6.09202C20.8324 6.19091 20.9078 6.39235 20.9521 6.93567C20.9984 7.50235 21 8.24689 21 9.4V14.6C21 15.7531 20.9984 16.4977 20.9521 17.0643C20.9078 17.6077 20.8324 17.8091 20.782 17.908C20.5903 18.2843 20.2843 18.5903 19.908 18.782C19.8091 18.8324 19.6077 18.9078 19.0643 18.9521C18.4977 18.9984 17.7531 19 16.6 19H7.4C6.24689 19 5.50235 18.9984 4.93567 18.9521C4.39235 18.9078 4.19091 18.8324 4.09202 18.782C3.71569 18.5903 3.40973 18.2843 3.21799 17.908C3.1676 17.8091 3.09225 17.6077 3.04785 17.0643C3.00156 16.4977 3 15.7531 3 14.6V9.4C3 8.24689 3.00156 7.50235 3.04785 6.93567C3.09225 6.39235 3.1676 6.19091 3.21799 6.09202C3.40973 5.71569 3.71569 5.40973 4.09202 5.21799C4.19091 5.1676 4.39235 5.09225 4.93567 5.04785C5.50235 5.00156 6.24689 5 7.4 5ZM1 9.4C1 7.15979 1 6.03969 1.43597 5.18404C1.81947 4.43139 2.43139 3.81947 3.18404 3.43597C4.03969 3 5.15979 3 7.4 3H16.6C18.8402 3 19.9603 3 20.816 3.43597C21.5686 3.81947 22.1805 4.43139 22.564 5.18404C23 6.03969 23 7.15979 23 9.4V14.6C23 16.8402 23 17.9603 22.564 18.816C22.1805 19.5686 21.5686 20.1805 20.816 20.564C19.9603 21 18.8402 21 16.6 21H7.4C5.15979 21 4.03969 21 3.18404 20.564C2.43139 20.1805 1.81947 19.5686 1.43597 18.816C1 17.9603 1 16.8402 1 14.6V9.4ZM6.5547 7.16795C6.09517 6.8616 5.4743 6.98577 5.16795 7.4453C4.8616 7.90483 4.98577 8.5257 5.4453 8.83205L11.4453 12.8321C11.7812 13.056 12.2188 13.056 12.5547 12.8321L18.5547 8.83205C19.0142 8.5257 19.1384 7.90483 18.8321 7.4453C18.5257 6.98577 17.9048 6.8616 17.4453 7.16795L12 10.7981L6.5547 7.16795Z"
-                    fill="white"
-                  />
-                </svg>
-              </a>
-            </li>
-          </ul>
-          <a href="#" class="menu-box">
-            <img class="menu-icon" src="./img/menu.png" alt="menu icon" />
-          </a>
-        </nav>
-      </div>
+        <a href="#" class="menu-box">
+          <img class="menu-icon" src="img/menu.png" alt="menu icon" />
+        </a>
```

Comment:
- [ OPTIONAL ] Great job so far @Alibaba2023 and @MasumaJaffery I will suggest that you make both the `mobile nav` and the `desktop nav ` fixed by using css to give it a `position: fixed`. This is because since they contain essential links such as `home`, `about`, `works`, `contact`, etc. By making it fixed, users can quickly access these important sections of the website without having to scroll back to the top or search for the navigation menu. 😉